packages/synapse-react-client/demo/containers/playground/AccessRequirementDemo.tsx

```diff
@@ -16,7 +16,7 @@ export const AccessRequirementDemo: React.FunctionComponent = () => {
   return (
     <div>
       {displayAccessRequirement ? (
-        <AccessRequirementListV2
+        <AccessRequirementList
```

Comment:
Old component was deleted, `AccessRequirementListV2` was renamed to `AccessRequirementList`